input/data/2006/2006ApJ...642L.119A/tev-000055.yaml

```diff
@@ -0,0 +1,27 @@
+source_id: 55
+reference_id: 2006ApJ...642L.119A
+telescope: magic
+data_entry_status: incomplete
+
+data:
+  livetime: 8.2h
+  significance: 6.4
+  excess: 560
+
+pos:
```

Comment:
Can you double check my entries for position and morph. Not completely clear to me. (page 3, left top).

Reply:
This is an AGN, a point source. It doesn't have a measured extension and the paper also doesn't give a position measurement. Fixed.